wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -612,12 +612,12 @@ data class NullTarget(
   ): SchemaHandler {
     return object : SchemaHandler {
       override fun handle(type: Type): Path? {
-        logger.artifactSkipped(type.type)
+        logger.artifactSkipped(type.type, "Dry-run")
```

Comment:
nit: `Dry-run` isn't in the documented set of targets. Would you be open any of the following ideas:

1. Refactor in `WireCompiler` to use a specialised `DryRunLogger`
2. Refactor `ConsoleWireLogger` to add a `isDryRun` field to match the `quiet` field

Reply:
Or maybe kill the dry-run feature... Wire Gradle plugin doesn't provide it.

How would the `DryRunLogger` work? Dry-runs use the target `NullTarget`, we'd need a way to know which target was replaced by this one

Reply:
I think we kill it from the core project. It’s trivial to do as a custom handler now.

Reply:
I'll drop it in follow up